modules/gamoshiBidAdapter.js

```diff
@@ -133,11 +150,20 @@ export const spec = {
           const playerSize = mediaTypes.video.playerSize || sizes;
           const videoImp = Object.assign({}, imp, {
             video: {
-              protocols: params.protocols || [1, 2, 3, 4, 5, 6],
+              // protocols: params.protocols || [1, 2, 3, 4, 5, 6],
+              protocols: bidRequest.mediaTypes.video.protocols || [1, 2, 3, 4, 5, 6],
```

Comment:
why not bidRequest.mediaTypes.video.protocols || params.protocols || [1, 2, 3, 4, 5, 6] in case you have legacy integrations only passing params